lapce-data/src/data.rs

```diff
@@ -156,20 +167,44 @@ impl LapceData {
     }
 }
 
+/// `LapceWindowData` is the application model for a top-level window,
+/// that is a window which can be independently moved around and
+/// resized using your window manager. Normally Lapce has only one
+/// top-level window, but new ones can be created using the "New Window"
+/// command.
+/// 
+/// Each window has its own collection of tabs (again, there is normally
```

Comment:
You might want to emphasize here as well that these tabs are not the commonly used editor tabs.

Reply:
Done.